package.json

```diff
@@ -25,6 +27,7 @@
     "@typescript-eslint/eslint-plugin": "^5.29.0",
     "@typescript-eslint/parser": "^5.29.0",
     "css-loader": "^6.7.1",
+    "cypress": "^9.7.0",
```

Comment:
Let's try to use the latest version`^11.0.1`

Reply:
if latest cypress version is used, we also need update Cypress structures/folders  @cyril-ui-developer

Reply:
> if latest cypress version is used, we also need update Cypress structures/folders since there are breaking changes from 9.7.0 to 10.x or 11.x @cyril-ui-developer

@yapei I need to verify from our team if we need to use what you are suggesting. 
@spadgett @jhadvig WDYT?